website/index.html

```diff
@@ -5,6 +5,26 @@
 </head>
 <body>
     <h1>Xbox controller panel</h1>
+   <ul>
+    <li><b>Left Trigger</b>:Move Backward</li>
+    <li><b>Left Bumper</b>:Belt</li>
+    <li><b>L3</b>:Preset 1</li>
```

Comment:
Maybe make this "Camera Preset 1" so it's a little more descriptive

Reply:
![image](https://github.com/user-attachments/assets/e45c8516-b9e2-4cf1-9ec3-c428619dee11)
Image of completed control guide.